### PR TITLE
[super-agent] Making sure all resources are deleted

### PR DESCRIPTION
**Pull request by paologallinaharbur**
#### Is this a new chart
This PR:
 -  forces `CT` to use a separate naspace so that we are sure `kubectl delete ns ...` will wait that resources are garbage collected (`ct` is not doing so when running helm delete)
 - modifies the uninstall job to make sure the super agent is NOT running before deleting resources. This is needed to avoid the SA recreating it right after  

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Review by sigilioso — APPROVED** [review]
LGTM 👍

File: .github/workflows/lint_test_charts.yaml [hunk 1]
```diff
@@ -86,8 +86,6 @@ jobs:
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
           start args: "--container-runtime=containerd"
-      - name: Create test ns
-        run: kubectl create ns ct
```
Comment (kang-makes) [inline .github/workflows/lint_test_charts.yaml:90]: The thing on testing in a namespace (`ct` in this pipeline) it to test that we are installing it a namespace that is neither `default` nor `newrelic` so you can test that all the objects are created in a different namespace and are able to boot.

I don't get why we need to remove the namespace. I don't get the race condition we are fixing... Why without it we remove the race condition?

Comment (paologallinaharbur) [reply .github/workflows/lint_test_charts.yaml:90]: Without `--namespace ns` selects a random namespace that it is then deleted:
```
namespace "super-agent-6s2970ilqu" deleted
```

By doing so we are:
 - covering both that we are installing in a namespace different from default nor newrelic
 - having "for free" a `kubectl delete --wait` it waits for all resources to be deleted (sadly we cannot add it as a flag of `Helm delete`)

Comment (kang-makes) [reply .github/workflows/lint_test_charts.yaml:90]: > selects a random namespace that it is then deleted

TIL. Sounds awesome then :D

**Review by kang-makes — APPROVED** [review]

File: .github/workflows/lint_test_charts.yaml (hunk 1 above)
Comment (juanjjaramillo) [reply .github/workflows/lint_test_charts.yaml:90]: Cool, I learned something new too

**Review by juanjjaramillo — APPROVED** [review]

**Pull request merged**